### PR TITLE
updated changelog

### PR DESCRIPTION
**Pull request by RingerJK**

### CODE REVIEW

**Comment by yunikkk** [comment]
cc @chezzdev

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.4.1
+
+### Vision
+- Changed behaviour on simultaneous `VisionManager` and `VisionReplayManager` instances creation to throwing an exception
```
Comment (yunikkk) [inline CHANGELOG.md]: Add `Fixed possible crash on VisionSafety/VisionArManager create/destroy`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+- Fixed `ArLane` in Replay mode
```
Comment (yunikkk) [inline CHANGELOG.md]: Be more descriptive? Eg. `Fixed 'ArLane' not displayed in VisionReplayManager`?

File: CHANGELOG.md (hunk 1 above)
Comment (chezzdev) [reply CHANGELOG.md]: Agree, we've put it like 
```
- Fixed a crash that may happen on creating or destroying `VisionARManager` or `VisionSafetyManager`
```

Comment (RingerJK) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (RingerJK) [reply CHANGELOG.md]: done

Comment (chezzdev) [reply CHANGELOG.md]: I think the reason for incorrect display of `ARLane` was incorrect values of `ARCamera` which could also affect the users. I would focus on this change and maybe mention lane issues as the consequence. In my understanding `ARLane` values were correct.

Comment (RingerJK) [reply CHANGELOG.md]: make sense

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.4.1
+
+### Vision
+- Changed behaviour on simultaneous `VisionManager`
+ and `VisionReplayManager` instances creation to throwing an exception
+- Fixed Replay mode obtained wrong camera-params (reason of incorrect work `ArLane`)
```
Comment (chezzdev) [inline CHANGELOG.md]:
```suggestion
- Fixed wrong `CameraParams` in replay mode (reason for incorrect AR lane display)
```

**Pull request merged**